src/shims/env.rs

```diff
@@ -36,6 +41,10 @@ impl<'tcx> EnvVars<'tcx> {
         }
         ecx.update_environ()
     }
+
+    pub(super) fn values(&self) -> InterpResult<'tcx, Values<'_, OsString, Pointer<Tag>>> {
```

Comment:
Why did you add this, and with `pub(super)`? Where is this called outside the current module?

Reply:
I used it for implementing shim `GetEnvironmentStringsW`([link to code](https://github.com/rust-lang/miri/pull/1225/files#diff-6d61240795974400d2a4ee29e5c02d39R133)). 

I wanted to use the least public specifier that lets the current PR compile, so that's how I added `pub(super)`.

Reply:
But that code is in the same module. It doesn't need this method, it can just directly access the `HashMap`.